src/styles/bootstrap.scss

```diff
@@ -105,4 +112,57 @@
 
     width: unset;
   }
+
+  .os-form-check {
+    /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
+    @extend .form-check;
+
+    padding-left: 0;
+  }
+
+  .os-form-check-input {
+    /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
+    @extend .form-check-input;
+
+    min-width: 1.2em;
+    min-height: 1.2em;
+    border: 2px solid #000;
+  }
+
+  .os-form-check-input[type="checkbox"] {
+    margin-right: .5rem;
+    margin-top: 0;
+  }
+
+  .os-form-check-input:focus {
+    /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
+    @extend .form-check-input;
+
+    min-width: 1.2em;
+    min-height: 1.2em;
+    box-shadow: 0 0 0 0.25rem #159db271;
+  }
+
+  .os-form-check-input:checked {
+    /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
+    @extend .form-check-input;
+
+    min-width: 1.2em;
+    min-height: 1.2em;
+    background-color: #159eb2;
+    border-color: #159eb2;
+  }
+
+  .os-btn {
+    /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
+    @extend .btn;
+
+    padding: 8px 32px;
+  }
+
+  .os-raise-bootstrap label {
```

Comment:
Given where this is, can't it just be?:

```css
  label {
    /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
    display: flex;
    align-items: center;
  }
```

It seems an implication of :point_up: is any label that is ever in `.os-raise-bootstrap` will get this styling. Should we be more selective to future proof ourselves a bit? For example, `CustomCheckbox` could support a prop that allows specifying classes to apply to the label. With that approach we can actually avoid having this here altogether and define a class in `interactives.scss`. :man_shrugging:

Reply:
I can't recall the exact reasoning that I wrote the above CSS, but it is not necessary to include in the `bootstrap.scss` file. Thanks for catching this.

Instead, I can add `display: flex` and `align-items: center` to `label.os-raise-fill-label-container` (maybe change the class name to `label.os-raise-label-container`) in `interactives.scss`. I think would resolve our issue and wouldn't require use of a prop in CustomCheckbox.